src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form class="form form_big" data-qa="big">
+      <label class="label label_big">
+        <input
+          class="input input_big"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </label>
+    </form>
+
+    <form class="form form_small" data-qa="small">
+      <label class="label label_small">
+        <input
+          class="input input_small"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+
   </body>
```

Comment:
```suggestion
    </form>
  </body>
```
Don't leave empty lines between parent and child elements